compiler/noirc_frontend/src/hir/resolution/resolver.rs

```diff
@@ -1512,19 +1512,27 @@ impl<'a> Resolver<'a> {
                     // Otherwise, then it is referring to an Identifier
                     // This lookup allows support of such statements: let x = foo::bar::SOME_GLOBAL + 10;
                     // If the expression is a singular indent, we search the resolver's current scope as normal.
-                    let (hir_ident, var_scope_index) = self.get_ident_from_path(path);
+                    let (hir_ident, var_scope_index) = self.get_ident_from_path(path.clone());
```

Comment:
Note: all of the changes in this file could be reverted, given that this code is now unused (the lsp code uses the elaborator). Should we undo this change?

Reply:
If it's not needed for this PR then I'd say yes

Reply:
Since it is already done, might as well keep it

Reply:
Oops, sorry, I just see this. I ended up removing it because it's unused and untested.